<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLZWindow</class>
 <widget class="QMainWindow" name="PLZWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Postleitzahlen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionSchliEen"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="actionNdern"/>
    <addaction name="actionLschen"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSchliEen">
   <property name="text">
    <string>Schli&amp;eßen</string>
   </property>
  </action>
  <action name="actionNeu">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionNdern">
   <property name="text">
    <string>Ändern</string>
   </property>
  </action>
  <action name="actionLschen">
   <property name="text">
    <string>Löschen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
